# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/nvidia/makeinst-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- NVIDIA-Linux-x86-1.0-4496-pkg2/Makefile.old	2003-07-29 13:40:28.000000000 +0200
+++ NVIDIA-Linux-x86-1.0-4496-pkg2/Makefile	2003-07-29 14:01:55.000000000 +0200
@@ -1,11 +1,4 @@
-default:
-	@ echo
-	@ echo "It is NOT recommended that you use this Makefile to install"
-	@ echo "the NVIDIA driver.  Instead, please use the nvidia-installer"
-	@ echo "utility.  However, should nvidia-installer not operate for you,"
-	@ echo "you may use this Makefile as a last resort.  To install using"
-	@ echo "this Makefile, please run 'make install'."
-	@ echo
+install: kernel_module_install install_gl
 
 kernel_module_install:
 	cd usr/src/nv; make install
@@ -21,7 +14,7 @@
 
 INSTALL_TLS := $(shell usr/bin/tls_test 2> /dev/null; if [ "$$?" = "0" ]; then echo "1"; else echo "0"; fi)
 
-install: kernel_module_install
+install_gl: 
 	@echo "Removing all old and conflicting files!"
 	@echo -n "      "
 	rm -f ${ROOT}/usr/lib/libGL.*
